feathr_project/feathr/api/api.py

```diff
@@ -0,0 +1,198 @@
+import os, json
+import logging
+from typing import List, Optional
+from fastapi import FastAPI, HTTPException, Response, status
+from pyapacheatlas.core import (AtlasException)
+from pydantic import BaseModel
+from opencensus.ext.azure.log_exporter import AzureLogHandler
+from azure.core.credentials import AzureKeyCredential
+from azure.core.exceptions import HttpResponseError, ServiceRequestError, ResourceNotFoundError
+from feathr.protobuf.featureValue_pb2 import FeatureValue
+from feathr._feature_registry import _FeatureRegistry
+from feathr._file_utils import write_to_file
+from feathr._synapse_submission import _FeathrSynapseJobLauncher
+from feathr._databricks_submission import _FeathrDatabricksJobLauncher
+from feathr.query_feature_list import FeatureQuery
+from feathr.settings import ObservationSettings
+from feathr.constants import *
+from feathr._feature_registry import _FeatureRegistry
+
+
+
+app = FastAPI()
+
+# Log Level
+log_level = os.getenv("logLevel", "INFO")
+
+logger = logging.getLogger(__name__)
+handler = logging.StreamHandler()
```

Comment:
move this into a common util.
I guess we should all use this

Reply:
Will address it after Yihui's PR gets in.

Reply:
Good feedback